File: 30daysOFcode_JavaScript/day 02.js
function operators({mealCost = 0, tipPercent = 0, taxPercent = 0}) {
    // Write your code here
    mealCost = Math.round(mealCost + (mealCost * tipPercent / 100) + (mealCost * taxPercent / 100));

    // Use console.log() to print to stdout
    console.log('The total meal cost is ' + mealCost + ' dollars.');
}

module.exports = operators;

File: 30daysOFcode_Ruby/day 02.rb
# Enter your code here. Read input from STDIN. Print output to STDOUT

meal = gets.to_f
tip = (gets.to_f / 100) * meal
tax = (gets.to_f / 100) * meal

total = (meal + tip + tax).round

puts "The total meal cost is #{ total } dollars."
